pkg/driver/driver.go

```diff
@@ -182,6 +182,12 @@ func (d *Driver) Run() error {
 	csi.RegisterNodeServer(d.Srv, d.NodeServer)
 
 	klog.Infof("Listening for connections on address: %#v", listener.Addr())
+
+	// Start taint watcher when gRPC server is ready to accept connections
+	if d.NodeServer.Clientset != nil {
```

Comment:
Would be cleaner if we just store `Clientset` in `Driver` itself without touching to `NodeServer`. We can also be sure that it is not nil and we can remove this if.